.github/workflows/update-page.yaml

```
@@ -0,0 +1,47 @@
+      repoName:
+        type: string
+        required: true
+        description: "Repository name?"
+
+jobs:
+  update-page:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # for updating index.yaml
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)